src/guidellm/benchmark/output.py

```diff
@@ -36,6 +37,8 @@
     safe_format_timestamp,
     split_text_list_by_length,
 )
+from guidellm.utils.dict import recursive_key_update
+from guidellm.utils.text import camelize_str
```

Comment:
```suggestion
from guidellm.utils import recursive_key_update, camelize_str
```

When importing from another submodule, import the top level